OpenRA.Game/Map/Map.cs

```diff
@@ -111,15 +111,33 @@ public enum MapVisibility
 		MissionSelector = 4
 	}
 
-	public class Map
+	public interface IMap
+	{
+		TileShape TileShape { get; }
+
+		int2 MapSize { get; set; }
+		bool Contains(CPos cell);
+		CPos CellContaining(WPos pos);
+		WVec OffsetOfSubCell(SubCell subCell);
+		IEnumerable<CPos> FindTilesInCircle(CPos center, int maxRange);
+		WPos CenterOfCell(CPos cell);
+	}
+
+	public class Map : IMap
 	{
 		public const int MaxTilesInCircleRange = 50;
 		public readonly TileShape TileShape;
+		TileShape IMap.TileShape
+		{
+			get { return TileShape; }
+		}
+
 		[FieldLoader.Ignore]
 		public readonly WVec[] SubCellOffsets;
 		public readonly SubCell DefaultSubCell;
 		public readonly SubCell LastSubCell;
-		[FieldLoader.Ignore] public IFolder Container;
+		[FieldLoader.Ignore]
+		public IFolder Container;
```

Comment:
We'll provide want to avoid pure formatting changes like this (i.e. moving the attribute to its own line), or keep them contained within their own commit at least.